RevenueCat/Editor/RevenueCatPostInstall.cs

```diff
@@ -7,52 +7,49 @@
 using UnityEditor.Callbacks;
 using UnityEditor.iOS.Xcode;
  
-namespace Editor
+public static class XcodeSwiftVersionPostProcess
```

Comment:
It might even be worth naming this `RevenueCatXcodePostProcess` to prevent further collisions

Reply:
not needed for this PR though